### PR TITLE
Fix misplaced braces in lint config

### LINKED ISSUE #30: invalid Configuration for rule 'no-unused-vars' in .eslintrc.json
tried to check with eslint, but the config seems wrong?
```
webui-src$ eslint app/*.js app/*/*.js

Oops! Something went wrong! :(

ESLint: 7.24.0

Error: .eslintrc.json:
        Configuration for rule "no-unused-vars" is invalid:
        Value {"args":"none","caughtErrors":"none","ignoreRestSiblings":true,"vars":"all","no-use-before-define":["warn",{"functions":true,"classes":false}],"object-shorthand":"error","prefer-const":["error",{"destructuring":"all"}],"semi-style":["warn","last"],"spaced-comment":["warn","always"]} should be equal to one of the allowed values.
        Value {"args":"none","caughtErrors":"none","ignoreRestSiblings":true,"vars":"all","no-use-before-define":["warn",{"functions":true,"classes":false}],"object-shorthand":"error","prefer-const":["error",{"destructuring":"all"}],"semi-style":["warn","last"],"spaced-comment":["warn","always"]} should NOT have additional properties.
        Value {"args":"none","caughtErrors":"none","ignoreRestSiblings":true,"vars":"all","no-use-before-define":["warn",{"functions":true,"classes":false}],"object-shorthand":"error","prefer-const":["error",{"destructuring":"all"}],"semi-style":["warn","last"],"spaced-comment":["warn","always"]} should match exactly one schema in oneOf.

    at ConfigValidator.validateRuleOptions (/usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:142:23)
    at /usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:197:18
    at Array.forEach (<anonymous>)
    at ConfigValidator.validateRules (/usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:194:34)
    at ConfigValidator.validateConfigArray (/usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:320:18)
    at CascadingConfigArrayFactory._finalizeConfigArray (/usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:493:23)
    at CascadingConfigArrayFactory.getConfigArrayForFile (/usr/lib/node_modules/eslint/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:299:21)
    at FileEnumerator._iterateFilesWithFile (/usr/lib/node_modules/eslint/lib/cli-engine/file-enumerator.js:365:43)
    at FileEnumerator._iterateFiles (/usr/lib/node_modules/eslint/lib/cli-engine/file-enumerator.js:346:25)
    at FileEnumerator.iterateFiles (/usr/lib/node_modules/eslint/lib/cli-engine/file-enumerator.js:296:59)

```

### PR DESCRIPTION
**Pull request by rottencandy**
Fixes #30

### CODE REVIEW

**Review by zeners — APPROVED** [review]
fixes #30

**Comment by rottencandy** [comment]
@zeners updated Readme with instructions.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,16 @@ References
 
 Contributing
 ------------
+Install `prettier` and `eslint` (required for linting & formatting code):
```
Comment (zeners) [inline README.md:75]: please add a caption like ### Code prepare

**Review by zeners — APPROVED** [review]
thx

**Pull request merged**